CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Control flow graph exceptions around empty statement lists.
 - Fallthrough successor being absent from `Cases` block.
+- Control flow graph construction for `finally` blocks inside `except` block. Started modelling
+  exceptions caught outside the `finally` block.
```

Comment:
We tend to use American English spellings in the documentation/changelog.
"modelling" -> "modeling"

Also, the extra detail in this entry is (imo) making it harder to understand.
It's about improving the CFG modeling around finally blocks that are nested within except blocks, yeah?